js/github.js

```diff
@@ -0,0 +1,27 @@
+// Pulls the last modified date of the current documentation from GitHub
+
+var fetch_github_date = function(callback) {
+    if (typeof github_repo == "undefined" || typeof github_path == "undefined")
+        return;
+
+    var url = "https://api.github.com/repos/" + github_repo + "/commits?path=" + github_path;
+    $.getJSON(url, function(data) {
+        if (data.length < 1)
+            return;
+
+        var commit = data[0];
+        var dateobj = new Date(commit.commit.committer.date);
+        var datefmt = dateobj.toLocaleString();
+
+        callback(datefmt);
+    });
+};
+
+$(function() {
+    if ($(".github-date").length > 0) {
```

Comment:
You should also check for the existence of $(".github-date-value") while you are here.